docs/getting-started/storing-data.mdx

```diff
@@ -339,4 +311,4 @@ git commit -m "add incrementor contract"
 
 In this section, we added a new contract to this project, reorganizing the project as a multi-contract project using Cargo Workspaces. The new contract made use of Soroban's storage capabilities to store and retrieve data. We also learned about the different kinds of storage and how to manage their lifetimes.
 
-Next we'll learn to deploy contracts to Soroban's Testnet network and interact with them over RPC using the CLI.
+Next we'll learn a bit more about deploying contracts to Soroban's Testnet network and interact with our incrementor contractC using the CLI.
```

Comment:
```suggestion
Next we'll learn how to deploy our incrementor contract to Soroban's Testnet network and interact with it using the CLI.
```

Reply:
I like the "a bit more about deploying" version better, since there was a previous step about deploying the one-step way, and the next page is about the two-step way.

Looks like there is a typo to be fixed, though. `contractC`